imap_processing/tests/hi/test_hi_l1b.py

```diff
@@ -142,3 +150,36 @@ def test_compute_coincidence_type_and_time_deltas(synthetic_trigger_id_and_tof_d
             np.bitwise_and(updated_dataset.coincidence_type, CoincidenceBitmap.C2),
         ),
     )
+
+
+@pytest.mark.parametrize("sensor_number", [45, 90])
+@mock.patch("imap_processing.hi.l1b.hi_l1b.instrument_pointing")
```

Comment:
This mocks out the `instrument_pointing` function so that SPICE isn't needed for this test.